Makefile

```diff
@@ -82,6 +82,9 @@ before_commit: code-style update-scenarios
 docker-build: ## Build Docker images that can be used for tests
 	docker build -t insights-behavioral-spec:ci .
 
+install-type-libraries: ## Install libraries with type definitions
```

Comment:
Shouldn't this be a pre-requisite for `type-checks` target?

Reply:
I was thinking about it. Then it will be run everytime.

Reply:
true, but if you already have the libraries `pip` will check that and exit fast